go/cmd/vtctldclient/internal/command/schema.go

```diff
@@ -29,12 +29,26 @@ import (
 	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
 )
 
-// GetSchema makes a GetSchema gRPC call to a vtctld.
-var GetSchema = &cobra.Command{
-	Use:  "GetSchema [--tables TABLES ...] [--exclude-tables EXCLUDE_TABLES ...] [{--table-names-only | --table-sizes-only}] [--include-views] alias",
-	Args: cobra.ExactArgs(1),
-	RunE: commandGetSchema,
-}
+var (
+	// GetSchema makes a GetSchema gRPC call to a vtctld.
+	GetSchema = &cobra.Command{
+		Use:  "GetSchema [--tables TABLES ...] [--exclude-tables EXCLUDE_TABLES ...] [{--table-names-only | --table-sizes-only}] [--include-views] alias",
+		Args: cobra.ExactArgs(1),
+		RunE: commandGetSchema,
+	}
+	// ValidateSchemaKeyspace makes a ValidateSchemaKeyspace gRPC call to a vtctld.
+	ValidateSchemaKeyspace = &cobra.Command{
+		Use:  "ValidateSchemaKeyspace [--include-vschema] [--skip-no-primary] [--include-views] [--exclude-tables TABLES ...] keyspace",
```

Comment:
nit: these should be written as `<keyspace>` not `keyspace` (ditto for `<keyspace/shard>` on the next one)